arrow.rdf

```diff
@@ -44,15 +50,12 @@
     <programming-language>R</programming-language>
     <programming-language>Ruby</programming-language>
     <programming-language>Rust</programming-language>
-    <category rdf:resource="http://projects.apache.org/category/library" />
-    <category rdf:resource="http://projects.apache.org/category/big-data" />
-    <release>
-      <Version>
-        <name>Apache Arrow</name>
-        <created>2022-10-31</created>
-        <revision>10.0.0</revision>
-      </Version>
-    </release>
```

Comment:
Can we keep `<release>`?
(The latest version is `13.0.0` at `2023-08-23`: https://arrow.apache.org/release/ )